src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -79,11 +73,14 @@ func NewClient(addr, appName string) *MobileClient {
 		}
 	}
 
-	analytics.SessionEvent(hqfdc, sessionPayload)
+	mch.MakeWithClient()(func(c *http.Client) {
+		analytics.SessionEvent(c, sessionPayload)
```

Comment:
If we don't really need to direct front to analytics, then we don't need the `mch` here and thus we don't need to store it in the `MobileClient` at all.